src/main/java/com/espot/employeemanagementapp/Entity/Employee.java

```diff
@@ -0,0 +1,29 @@
+package com.espot.employeemanagementapp.Entity;
```

Comment:
package name Entity should be small case, `package com.espot.employeemanagementapp.entity;`